### PR TITLE
Adding support for multiple replyTo values

### PR DESCRIPTION
**Pull request by avivshafir**
Following this issue: #118

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @avivshafir,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by Mogztter** [comment]
Thanks for your contribution.

I think it could be nice to stay backward compatible. Could you please preserve the `setReplyTo` method (with a deprecated tag and a comment to say please use `addReplyTo` instead).
The `setReplyTo` method will assign a single email to the list.

**Comment by Mogztter** [comment]
Reading the code again, I'm not sure this is a good idea... Since we are breaking the API we will need to release a major version.

File: src/main/java/play/libs/mailer/Email.java [hunk 1]
```diff
@@ -99,11 +99,16 @@ public Email setBcc(List<String> bcc) {
     return this;
   }
 
-  public String getReplyTo() {
+  public List<String> getReplyTo() {
```
Comment (mkurz) [inline src/main/java/play/libs/mailer/Email.java]: @Mogztter We also would need to deprecate this one and instead add something like `public List<String> getRepliesTo() { ... }` or `getAllReplyTo()`.
The question though is what the deprecated `String getReplyTo` should return? Just the first email in the list (if there is one)?

**Comment by mkurz** [comment]
In case we really break backwards compatibility we would release the new major version along with Play 2.6 which will released within the next couple weeks (RC1 is planned in two weeks).

**Comment by mkurz** [comment]
So assuming we release a new major version next (along with Play 2.6) we could merge this even though it breaks compatibility. WDYT?

**Comment by mkurz** [comment]
@Mogztter @marcospereira How should we continue?

**Comment by Mogztter** [comment]
I think it's OK to break compatibility along with Play 2.6.
Since master is already based on Play 2.6, I think it's safe to merge this pull request and later release a new milestone (6.0.0-M2) with this feature.

Does this sound good ?

**Comment by mkurz** [comment]
Sounds good to me :+1:

**Comment by marcospereira** [comment]
@avivshafir could you please rebase against master and fix the conflicts.

After that, I think this would be ok to be merged.

Thanks.

**Pull request merged**